cmd/clusterctl/examples/vsphere/generate-yaml.sh

```diff
@@ -129,6 +136,41 @@ if [ -z $VSPHERE_MANAGER_IMG ]; then
   exit 1
 fi
 
+if [ -z $VSPHERE_DATACENTER ]; then
+  echo "env var VSPHERE_DATACENTER is required"
```

Comment:
Hi @andrewsykim,

Anywhere `echo` is used to indicate an error, please use `echo "your message" 1>&2` to ensure the message is printed to `stderr`. 

Please note this applies to several other lines, but I won't repeat this comment there.